cinema/serializers.py

```diff
@@ -1 +1,98 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import (
+    Actor,
+    Genre,
+    CinemaHall,
+    Movie,
+    MovieSession,
+)
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Actor
+        fields = (
+            "id", "first_name",
+            "last_name", "full_name",
```

Comment:
hmm, did you run `black`? It doesn't look good to me, but I can't tell that it's a mistake

Reply:
just maybe 
1) 
```python
fields = ("a", "b", "c", "d")
```
or 
2) 
```python
fields = (
"a",
"b",
"c",
"d"
)
```
would that be better? WDYT?

Reply:
I agree with you, the second option will be better in this case)